src/lib/ThumbnailsSidebar.js

```diff
@@ -81,15 +84,38 @@ class ThumbnailsSidebar {
             const { bpPageNum: pageNumStr } = thumbnailEl.dataset;
             const pageNum = parseInt(pageNumStr, 10);
 
-            if (this.onClickHandler) {
-                this.onClickHandler(pageNum);
+            if (this.onThumbnailSelect) {
+                this.onThumbnailSelect(pageNum);
             }
         }
 
+        this.anchorEl.focus();
         evt.preventDefault();
         evt.stopImmediatePropagation();
     }
 
+    /**
+     * Method to handle keyboard events in the Thumbnails Sidebar
+     *
+     * @param {Object} evt - Key event
+     * @return {void}
+     */
+    onKeydown(evt) {
+        const key = decodeKeydown(evt);
+        let nextSelectedPage = this.currentPage;
+        if (key === 'ArrowUp') {
```

Comment:
What happens if we're on the first or last page and hit up arrow or down arrow, respectively?

Reply:
Nothing will happen because we check `nextSelectedPage !== this.currentPage`

Reply:
setPage in docBase Viewer will prevent the keydown from changing anything.